portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,28 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  
+  private ArrayList<String> messages;
+  
+  public void init() {
+    messages = new ArrayList<String>();
+    messages.add("Lovely weather we\'re having");
+    messages.add("How was your day?");
+    messages.add("Honey, I\'m home!");
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Meagan!</h1>");
+    init();
```

Comment:
This doesn't need to be called explicitly. It should be called during servlet initialization.